### PR TITLE
Add typings for jsonminify

### PR DESCRIPTION
**Pull request by no23reason**
Added typings for [jsonminify](https://www.npmjs.com/package/jsonminify).

### CODE REVIEW

File: jsonminify/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+// Definitions by: Dan Homola <https://github.com/no23reason>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface MinifyFn {
+    (json: string): string;
+}
+
+declare var minify: MinifyFn;
+
+declare module "jsonminify" {
```
Comment (vvakame) [inline jsonminify/index.d.ts]: could you strip `declare module "jsonminify" {`?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred

Comment (no23reason) [inline jsonminify/index.d.ts]: Updated

File: jsonminify/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// Type definitions for jsonminify 0.4.1
+// Project: https://github.com/fkei/JSON.minify
+// Definitions by: Dan Homola <https://github.com/no23reason>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface MinifyFn {
+    (json: string): string;
+}
+
+declare var minify: MinifyFn;
```
Comment (andy-ms) [inline jsonminify/index.d.ts]: This could just be `declare function minify(json: string): string;`.

Comment (no23reason) [inline jsonminify/index.d.ts]: Thank you @andy-ms, you are right, I overly complicated it. I've fixed the declaration.

**Review by ghost — APPROVED** [review]

File: jsonminify/index.d.ts [hunk 3]
```diff
@@ -3,6 +3,6 @@
 // Definitions by: Dan Homola <https://github.com/no23reason>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare function minify(json: string): string;
+declare const minify: (json: string) => string;
```
Comment (no23reason) [inline jsonminify/index.d.ts]: The simplification you suggested @andy-ms turned out to break the CI (I apologise for committing it without thoroughly testing it). This version should be finally ok.

Comment (andy-ms) [inline jsonminify/index.d.ts]: The problem is that the tests are using `import * as jsonminify from "jsonminify";`, an ES6-style import.
jsonminify is a CommonJS module, so you need to import it using `import jsonminify = require("jsonminify")`.

Comment (no23reason) [inline jsonminify/index.d.ts]: It makes sense now! Thank you very much for your time and sorry it took so many commits... It's fixed now.

**Pull request merged**